integtest/run.bash

```diff
@@ -81,7 +82,7 @@ initialize_k8s_master() {
     kubectl create --filename https://docs.projectcalico.org/manifests/tigera-operator.yaml
     kubectl create --filename https://docs.projectcalico.org/manifests/custom-resources.yaml
 
-    kubectl wait "node/${EXOSCALE_MASTER_NAME}" --for=condition=Ready --timeout=180s
+    kubectl wait "node/${EXOSCALE_MASTER_NAME}" --for=condition=Ready --timeout=600s
```

Comment:
Might be worth storing this value in a global shell variable/constant and reference it in all the commands instead 😏